docs/workflow.md

```diff
@@ -118,6 +119,7 @@ The Pull Request process usually looks like the following:
           2. Leave comments unresolved, let the reviewer resolve them.
        2. Don't be afraid to ask for clarification regarding changes or suggest alternatives if you don't agree with what was suggested. The reviewers and reviewee should work together to come up with the best solution.
        3. **Do not resolve conversations as you address them** (but make sure to leave a comment as mentioned above). That is the responsibility of the reviewers.
+       4. Once you have addressed all the comments, re-request review from reviewers.
 11. Make sure our automated tests with Travis CI are passing. There will be an indicator near the bottom of the Pull Request. If something fails, you can click on the links provided to get more information and debug the problems. More than likely, you'll just need to re-run clang-format on the code.
```

Comment:
Maybe add in that if CI times out for whatever reason, you'll need to log in to TravisCI and re-run it.

Reply:
restarting CI is not part of the normal workflow and we would like members to report that if it happens